src/components/TransactionBottomBar.tsx

```diff
@@ -43,14 +61,50 @@ export const ClearResults: React.FC<{ type: ResultType }> = ({ type }) => {
 };
 
 const TransactionBottomBar: React.FC = () => {
+
+	const { x, y } = useMousePosition();
+
+  const [resultHeight, setResultHeight] = useState(140);
+  const [isResizingResult, setIsResizingResult] = useState(false);
+
+
+  const toggleResizingResult = (toggle: boolean) => {
+    setIsResizingResult(toggle);
+  };
+
+  const toggleResizeListener = () => {
+    toggleResizingResult(false);
+  };
+
+  useEffect(() => {
+    if (
+      isResizingResult &&
+      y > RESULT_PANEL_MIN_HEIGHT &&
+      y < window.innerHeight - PLAYGROUND_HEADER_HEIGHT
+    ) {
+      setResultHeight(y);
+    }
+  }, [x, y]);
+
+  useEffect(() => {
+    window.addEventListener('mouseup', toggleResizeListener, false);
+    return () => {
+      window.removeEventListener('mouseup', toggleResizeListener, false);
+    };
+  }, []);
+
+
   return (
+		<FeedbackContainer height={resultHeight}>
     <FeedbackRoot>
-      <Heading>
+		<ResizeHeading onMouseDown={() => toggleResizingResult(true)}>
         Transaction Results
         <ClearResults type={ResultType.Transaction} />
-      </Heading>
+			<div></div>
```

Comment:
...and the last one :)

Reply:
@MaxStalker So these are 'placeholder' divs that force the layout. Later I want to add a button to minimize and maximize the panels, where these divs are now.